### PR TITLE
Update github issue templates

### PR DESCRIPTION
**Pull request by JustinBeckwith**

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+---
+
+Thanks for stopping by to let us know something could be better!
+
+**PLEASE READ**: If you have a support contract with Google, please create an issue in the [support console](https://cloud.google.com/support/) instead of filing on GitHub. This will ensure a timely response.
+
+Please run down the following list and make sure you've tried the usual "quick fixes":
+
+  - Search the issues already opened: https://github.com/googleapis/google-api-ruby-client/issues
+  - Search StackOverflow: http://stackoverflow.com/questions/tagged/google-cloud-platform+ruby
```
Comment (blowmage) [inline .github/ISSUE_TEMPLATE/bug_report.md]: I think https://stackoverflow.com/questions/tagged/google-api-client+ruby would be a better StackOverflow link. Most APIs in the gem are not Cloud APIs.

Comment (JustinBeckwith) [reply .github/ISSUE_TEMPLATE/bug_report.md]: Fixed!

**Comment by JustinBeckwith** [comment]
@TheRoyalTnetennba the kokoro tests are revolting

**Comment by TheRoyalTnetennba** [comment]
lol the PR containing the configs hasn't been merged yet.

**Pull request merged**